lib/Dialect/Torch/IR/TorchOps.cpp

```diff
@@ -763,6 +763,43 @@ void AtenLenTOp::getCanonicalizationPatterns(RewritePatternSet &patterns,
   });
 }
 
+//===----------------------------------------------------------------------===//
+// AtenAddTensorOp
+//===----------------------------------------------------------------------===//
+
+void AtenAddTensorOp::getCanonicalizationPatterns(RewritePatternSet &patterns,
+                                                  MLIRContext *context) {
+  patterns.add(+[](AtenAddTensorOp op, PatternRewriter &rewriter) {
+    // The lhs and rhs of the add.tensor op should be 0d tensors for the
+    // canonicalization to be carried out.
+    // `aten.add.tensor(self, other, alpha)` is canonicalized to
+    // `aten.add.int(self, aten.mul.int(other, alpha))`.
+
+    ValueTensorLiteralOp lhsScalarValueTensor = op.self().getDefiningOp<ValueTensorLiteralOp>();
```

Comment:
This is too specific. We should have a common helper to get the scalar Value (or null) from a tensor Value.

Reply:
@silvasean Sure, I will handle that after I solve the issue regarding extraction value from ElementsAttr.